R/commas_linter.R

```diff
@@ -15,6 +15,10 @@ commas_linter <- function() {
 
     res <- re_matches(source_expression$lines, re, global = TRUE, locations = TRUE)
 
+    if (length(res) == 1L && nrow(res[[1L]]) == 1L && all(is.na(res[[1L]]))) {
```

Comment:
`commas_linter()` was also affected by a similar issue to #1160; the fix here is easy